Using this prevents an error around line 220 below it.
"AttributeError: module 'importlib' has no attribute 'machinery'"
The old backup logic is in place and compiles without issues.
--- a/sources/shiboken2/libshiboken/embed/signature_bootstrap.py	2023-04-02 23:09:34.688912402 -0400
+++ b/sources/shiboken2/libshiboken/embed/signature_bootstrap.py	2023-04-02 23:10:03.395529582 -0400
@@ -170,9 +170,6 @@

     # 'zipstring_sequence' comes from signature.cpp
     zipbytes = base64.b64decode(''.join(zipstring_sequence))
-    if sys.version_info[0] >= 3:
-        vzip = zipfile.ZipFile(io.BytesIO(zipbytes))
-        return sys.meta_path, EmbeddableZipImporter(vzip)

     # Old version for Python 2.7, only.
     import os
